### ISSUE TITLE
in ios routes are not displaying

### ISSUE THREAD

**Issue by VakaGopi** [TorbenK/TK.CustomMap#209]
Hi,

 Thanks for develop this package. In my ios simulator routes are displaying but coming to device routes are not displaying.

Regards.

**Comment by TorbenK** [comment]
Do you call `TKCustomMapRenderer.InitMapRenderer();` ?

**Comment by VakaGopi** [comment]
yes i called it in my AppDelegate.cs

**Comment by VakaGopi** [comment]
In my ios simulator routes are displaying good but coming to device routes are not displaying.

please help me, it is so important to me

**Comment by TorbenK** [comment]
Maybe it is the linker again. You can try this. 

https://github.com/TorbenK/TK.CustomMap/issues/29#issuecomment-188826339

**Comment by VakaGopi** [comment]
no it's not working in my case. i don't have any problem in android and ios simulators also, routes are showing well but in ios devices only the routes were not showing

**Comment by TorbenK** [comment]
That sounds exactly like the linker issue to me. 

Anyway, you can observe `RouteCalculationFailed` to see what's going wrong.

**Comment by VakaGopi** [comment]
in ios simulator it's not firing, but in device it's firing and shows  error message as a "Directions Not Available".

**Comment by TorbenK** [comment]
Please check whether directions API is avaiable in your country. http://www.apple.com/uk/ios/feature-availability/#maps-directions

**Comment by VakaGopi** [comment]
yes directions API is not available here sorry

**Comment by filaulet** [comment]
Hi. I have the same problem. I tried the solution explained in post https://github.com/TorbenK/TK.CustomMap/issues/29#issuecomment-188826339: I inserted `var dummy = new TKCustomMapRenderer();` in AppDelegate.cs but it's not working for me. 
I also tried to observe `RouteCalculationFailed` and `RouteCalculationFinished` but neither of the two events seems to be intercepted. 
For Android, route is working.
I'm debugging on iPhone 5s and I have TK.CustomMap v2.0.0 and Xamarin.Forms.Maps v2.5.0.121934.
